### PR TITLE
[DOC] Update explanation of router service `isActive` method and add code example

### PR DESCRIPTION
**Pull request by pauljeannot**
Part of #16518

Took the [nice work done](https://github.com/emberjs/ember.js/pull/17622) by @rscotchmer, and updated it with new class syntax.

### CODE REVIEW

File: packages/@ember/-internals/routing/lib/services/router.ts [hunk 1]
```diff
@@ -244,12 +244,33 @@ export default class RouterService extends Service {
...
+       @service router;
+
+       displayComments() {
+         return this.get('router').isActive('posts');
+       }
+     });
+     ```
+
+     Usage example  with a model:
+     ```javascript return this.get('router').isActive('posts', post.id)```
```
Comment (rwjblue) [inline packages/@ember/-internals/routing/lib/services/router.ts]: I think we need some newlines here (after "javascript" and before the ending backticks at least)

File: packages/@ember/-internals/routing/lib/services/router.ts [hunk 2]
```diff
@@ -244,12 +244,33 @@ export default class RouterService extends Service {
   }
 
   /**
-     Determines whether a route is active.
+     Returns `true` if `routeName/models/queryParams` is the active route, where `models` and `queryParams` are optional.
+     See [model](api/ember/release/classes/Route/methods/model?anchor=model) and
+     [queryParams](/api/ember/3.7/classes/Route/properties/queryParams?anchor=queryParams) for more information about these arguments.
+
+     In the following example, isActive will return `true` if the current route is `/posts`.
+     ```app/components/posts.js
```
Comment (locks) [inline packages/@ember/-internals/routing/lib/services/router.ts]:
```suggestion

     ```app/components/posts.js
```

File: packages/@ember/-internals/routing/lib/services/router.ts (hunk 1 above)
Comment (locks) [inline packages/@ember/-internals/routing/lib/services/router.ts]:
```suggestion
     
     ```javascript
     return this.get('router').isActive('posts', post.id)
     ```
```

Comment (locks) [reply packages/@ember/-internals/routing/lib/services/router.ts]: We also try to make the code samples as complete as possible. Can you add some more context to it so it's easier to understand where it goes?

**Review by locks — CHANGES_REQUESTED** [review]
Left some inline asks :)

**Review by pzuraq — COMMENTED** [review]
This generally looks good, just need to fix the formatting around the examples (per @locks suggestions). More context for the examples would be good as well.

**Comment by locks** [comment]
Thank you for your help!

**Pull request merged**